modules/dag-l1/src/main/scala/org/tessellation/dag/l1/StateChannel.scala

```diff
@@ -160,7 +160,7 @@ class StateChannel[F[_]: Async: KryoSerializer: SecurityProvider: Random](
     }
 
   private val sendBlockToL0: Pipe[F, FinalBlock, FinalBlock] =
-    _.evalTap { fb =>
+    _.evalFilter { fb =>
```

Comment:
If sending to L0 failed, it's not a reason to discard such a block from further processing. Two other L1 facilitators may still send the block successfully to L0. Additionally we should rather optimize process of sending to L0 (retries?) then discard the block if sending failed.

Reply:
Code reverted.